ystafdb/config_parser.py

```diff
@@ -1,13 +1,17 @@
+import os
 import json
-
+from pathlib import Path
 
 def get_config_data():
     providers = []
     datasets = []
     providersRequiredFields = set()
     datasetsRequiredFields = set()
 
-    with open('ystafdb/data/config.json') as json_file:
+    if not os.path.exists(Path("config.json")):
```

Comment:
are we sure this checks the *current* directory where the script is run?

Reply:
os.path.exists checks a path relative to where the script is run. 
Therefor this will only work if the script is run from the root project folder.

Should it be made more generic?